src/components/sheet/SheetGestureBlocker.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ScrollView, View } from 'react-native';
+import { PanGestureHandler } from 'react-native-gesture-handler';
+import { Box } from '@/design-system';
+import { IS_IOS } from '@/env';
+
+export const SheetGestureBlocker = ({
+  children,
+  disabled,
+  preventScrollViewDismissal = true,
+}: {
+  children: React.ReactNode;
+  disabled?: boolean;
+  preventScrollViewDismissal?: boolean;
+}) => {
+  return IS_IOS ? (
```

Comment:
id love to add something to the component name that is like IOS related cause this shit always trips me up if i have no context and its not doing anything on android